code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -528,6 +425,48 @@ client/proc/one_click_antag()
 
 	return 0
 
+/datum/admins/proc/makeRaiders()
+	var/list/mob/candidates = list()
+	var/mob/theghost = null
+	var/input = "Loot and burn the station or trade with it."
+
+	for(var/mob/G in respawnable_list)
+		if(!jobban_isbanned(G, SPECIAL_ROLE_RAIDER))
+			spawn(0)
+				candidates = pollCandidates("Do you wish to be considered for a Space Pirate party about to be sent in?", SPECIAL_ROLE_RAIDER, 1)

```

Comment:
This is going to ask for candidates many times for the same person. Also, since the assignment to `candidates` is in a `spawn`, you can't use the returned value. You can just get rid of all the conditionals, spawn, and looping, and you'll be just fine.